### PR TITLE
feat(google-backstory): new library google-backstory 

### PR DESCRIPTION
**Pull request by suztomo**
I manually edit librarian.yaml and librarian.state file and ran `$ docker run -u $(id -u):$(id -g) -v .:/repo -v ~/.cache:/.cache -w /repo   docker.io/library/librarian-python:${V}  generate -v google-backstory`.

I referenced the google-cloud-common declaration in librarian.yaml. At first, I hit `2026/06/04 14:33:45 librarian: generate library "google-backstory" (python): error creating metadata for google-backstory: default version must be specified for every library with generated APIs` error but this was resolved by declaring `default_version: apiVersion`.

### Unit Test Failure and Fix
The CI check failed because `google-backstory` is a type-only library and had no unit tests, causing `pytest` to exit with code 5 (no tests collected).

To fix this, I added:
- `test_packaging.py` to verify namespace package compatibility.
- `test_backstory.py` to verify that backstory types can be imported.

This follows the pattern of `google-cloud-common` which also has a dummy test to avoid empty test suite failures. I also updated `librarian.yaml` (`keep`) and `.librarian/state.yaml` (`preserve_regex`) to preserve these tests during regeneration.

### CODE REVIEW

File: packages/google-backstory/google/backstory/types/collection.py [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+
+import google.protobuf.duration_pb2 as duration_pb2  # type: ignore
+import google.protobuf.struct_pb2 as struct_pb2  # type: ignore
+import google.protobuf.timestamp_pb2 as timestamp_pb2  # type: ignore
+import google.type.interval_pb2 as interval_pb2  # type: ignore
+import proto  # type: ignore
+
+from google.backstory.types import entity as gb_entity
+from google.backstory.types import id as gb_id
+from google.backstory.types import udm
```
[the remark this answers is not in the document]
Comment (suztomo) [reply packages/google-backstory/google/backstory/types/collection.py:28]: They are generated files.

File: packages/google-backstory/google/backstory/__init__.py [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+            parts = []
+            for part in version_string.split("."):
+                try:
+                    parts.append(int(part))
+                except ValueError:
+                    # If it's a non-numeric part (e.g., '1.0.0b1' -> 'b1'), stop here.
+                    # This is a simplification compared to 'packaging.parse_version', but sufficient
+                    # for comparing strictly numeric semantic versions.
+                    break
+            return tuple(parts)
```
[the remark this answers is not in the document]
Comment (suztomo) [reply packages/google-backstory/google/backstory/__init__.py:181]: They are generated files.

File: packages/google-backstory/google/backstory/__init__.py [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+                FutureWarning,
+            )
+    except Exception:
+        warnings.warn(
+            "Could not determine the version of Python "
+            + "currently being used. To continue receiving "
+            + "updates for {_package_label}, ensure you are "
+            + "using a supported version of Python; see "
+            + "https://devguide.python.org/versions/"
+        )
```
[the remark this answers is not in the document]
Comment (suztomo) [reply packages/google-backstory/google/backstory/__init__.py:221]: They are generated files.

File: packages/google-backstory/setup.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+dependencies = [
+    "google-api-core[grpc] >= 2.17.1, <3.0.0",
+    # Exclude incompatible versions of `google-auth`
+    # See https://github.com/googleapis/google-cloud-python/issues/12364
+    "google-auth >= 2.14.1, <3.0.0,!=2.24.0,!=2.25.0",
+    "grpcio >= 1.59.0, < 2.0.0",
+    "grpcio >= 1.75.1, < 2.0.0; python_version >= '3.14'",
+    "proto-plus >= 1.22.3, <2.0.0",
+    "proto-plus >= 1.25.0, <2.0.0; python_version >= '3.13'",
```
[the remark this answers is not in the document]
Comment (suztomo) [reply packages/google-backstory/setup.py:49]: They are generated files.

File: packages/google-backstory/setup.py [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+name = "google-backstory"
+
+
+description = "Google Backstory API client library"
+
+version = None
+
+with open(os.path.join(package_root, "google/backstory/gapic_version.py")) as fp:
+    version_candidates = re.findall(r"(?<=\")\d+.\d+.\d+(?=\")", fp.read())
```
[the remark this answers is not in the document]
Comment (suztomo) [reply packages/google-backstory/setup.py:32]: They are generated files.

File: packages/google-backstory/google/backstory/types/entity.py [hunk 6]
```diff
@@ -0,0 +1,976 @@
...
+from __future__ import annotations
+
+from typing import MutableMapping, MutableSequence
+
+import google.protobuf.struct_pb2 as struct_pb2  # type: ignore
+import google.protobuf.timestamp_pb2 as timestamp_pb2  # type: ignore
+import google.type.interval_pb2 as interval_pb2  # type: ignore
+import proto  # type: ignore
+
+from google.backstory.types import entity_risk, udm
```
[the remark this answers is not in the document]
Comment (suztomo) [reply packages/google-backstory/google/backstory/types/entity.py:25]: They are generated files.

**Review by parthea — APPROVED** [review]
